app/src/main/java/com/github/se/polyfit/ui/screen/IngredientsOverview.kt

```diff
@@ -23,11 +28,11 @@ import com.github.se.polyfit.viewmodel.qrCode.BarCodeCodeViewModel
 // TODO THIS CLASS IS TEMPORARY, REMOVE IT WHEN DOING THE VIEWMODEL
 data class IngredientsTMP(
     val name: String,
-    val servingSize: Int,
-    val calories: Int,
-    val carbs: Int,
-    val fat: Int,
-    val protein: Int
+    val servingSize: Double,
+    val calories: Double,
+    val carbs: Double,
+    val fat: Double,
+    val protein: Double
```

Comment:
Rename the class. TMP mean temporary, it was just for showing the data but if you keep it, change the name